data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/request/AuditErrorMessageTest.java

```diff
@@ -13,18 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.gov.gchq.palisade.service.data.request;
```

Comment:
Should be in the unit test package

Reply:
Decided component because spring json context stuff